File: src/data4help/src/main/resources/src/app/_models/blood-type.ts
export enum BloodType {
    A_POSITIVE,
    A_NEGATIVE,
    B_POSITIVE,
    B_NEGATIVE,
    AB_POSITIVE,
    AB_NEGATIVE,
    ZERO_POSITIVE,
    ZERO_NEGATIVE
}

export namespace BloodType {

    export function values() {
        return Object.keys(BloodType).filter(
            (type) => isNaN(<any>type) && type !== 'values'
        );
    }
}

File: DeliveryFolder/src/data4help/src/main/java/avila/schiatti/virdi/model/data/BloodType.java
package avila.schiatti.virdi.model.data;

import xyz.morphia.annotations.Embedded;

@Embedded
public enum BloodType {
    A_POSITIVE, A_NEGATIVE, B_POSITIVE, B_NEGATIVE, AB_POSITIVE, AB_NEGATIVE, ZERO_POSITIVE, ZERO_NEGATIVE
}
